11-multi-track/bb_views_part1/readme.md

```diff
@@ -172,14 +179,14 @@ And the in our index.html:
 
 ### Exercise: Define a Handlebars Template (10 minutes)
 
-1. Define a handlebars template for our grumble. (See the sample HTML below)
+1. Define a Handlebars template for our Grumble. See the sample HTML below.  
 2. On initialize, set a `this.template` property, which should point to a
-**compiled handlebars template**
+**compiled handlebars template**.  
 3. Update the `render` function to use the template. When using the template,
-we need to pass in just our model attributes, not the whole model, do some
-research to find the best way!
+we need to pass in just our model attributes - not the whole model. Do some
+research to find the best way!  
```

Comment:
So we need to pass `this.template` to `.html`. But what needs to be passed into `this.template`? (The model, but how do we write that out?)